flake.nix

```diff
@@ -27,17 +31,48 @@
           devShells.default = pkgs.mkShell {
             name = "hyperswitch-shell";
             packages = with pkgs; [
+              fd
```

Comment:
We don't necessarily _need_ `fd` as of now, we can add it later if required.

Reply:
FWIW, these are also included in `nixos-unified-template`

https://github.com/juspay/nixos-unified-template/blob/ed4bf673acdc802d06bfdd726494323ee2036f50/modules/home/packages.nix#L7-L12

Reply:
Removed this for now, can be added later if needed!